Cargo.toml

```diff
@@ -24,6 +24,7 @@ float_eq = "0.7"
 smallvec = "1.7.0"
 once_cell = "1.9.0"
 symphonia = { version = "0.4.0", default-features = false }
+delegate-attr = {git = "https://github.com/upsuper/delegate-attr.git", rev ="32cda21a40f6e8a1b069d6178455318d8ecd6803" }
```

Comment:
Why not just `delegate-attr = "0.2.9"`
I'd rather not take in dependencies that are not from crates.io

Reply:
It's a temporary one that I copied and pasted from another project of mine, I was going to remove the delegate-attr when I sent the PR since it's not needed if you don't want to hide BaseAudioContext from the crate users.